### ISSUE TITLE
First column for the back of counters

### ISSUE THREAD

**Issue by lifelike** [lifelike/countersheetsextension#14]
There is something confusing in how to layout the data CSV for double-sided counters. It has always been like that. Usually the first column after BACK must be empty, or at least it is ignored. It is probably an off-by-one bug rather than something that was done deliberately.

It can't be used exactly like the first column of the front, because that says how many counters to generate, and it must be the same for front and back. The header for that column also gives the id of a template to use for both front and back of the counter. Possibly because of that special column the first column for the back of the counter is also treated in a special way, which I think just means it is ignored (at least I can't remember anything it can be used for).

Suggested fix is to keep things as they are, with the header for the first column applying to front and back. It is possible to leave that blank anyway and use other columns to add templates that are specific for the front or back. But the first column after BACK is kinda useless. If it is wanted to add a template that can be done anyway (just say +templateid in the header for that column) so no need to try to reserve it for a special use.

tl;dr: Remove the required empty column to the right of the BACK column.

**Comment by lifelike** [comment]
Already working. Tested a few sheets and the empty column was ignored. Works as intended now. Good enough then.
